plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/order-summary/index.tsx

```diff
@@ -1,10 +1,13 @@
 /**
  * External dependencies
  */
-import { useContainerWidthContext } from '@woocommerce/base-context';
+import {
+	useContainerWidthContext,
+	useStoreCart,
+} from '@woocommerce/base-context';
 import type { CartItem } from '@woocommerce/types';
 import clsx from 'clsx';
-
+import { CartLineItemsSkeleton } from '@woocommerce/base-components/skeleton/patterns/cart-line-items';
```

Comment:
the skeleton components will be cleaned up in another PR

Reply:
There are a lot of components defined that we don't need. I also had to redo some of them. As I work through the Cart block, I will see what is needed and remove the rest of the skeleton patterns. A good overview of the current Skeletons is on Stylebook.